contributing/documentation/standards.rst

```diff
@@ -160,6 +172,7 @@ English Language Standards
 .. _`the Sphinx documentation`: http://sphinx-doc.org/rest.html#source-code
 .. _`Twig Coding Standards`: http://twig.sensiolabs.org/doc/coding_standards.html
 .. _`American English`: http://en.wikipedia.org/wiki/American_English
+.. _`American English Oxford Dictionary`: http://www.oxforddictionaries.com/
```

Comment:
This page doesn't seem like the best choice to me. At least, when I search for words there, I am asked to purchase a subscription to use their service.